packages/ens-utils/src/registrar.ts

```diff
@@ -175,29 +191,50 @@ export interface Registrar {
 
   getOnchainRegistrar(): ContractRef | null;
 
-  canRegister(name: ENSName, atTimestamp: Timestamp, duration: Duration, existingRegistration?: Registration): boolean;
-  getRegistrationPriceQuote(name: ENSName, atTimestamp: Timestamp, duration: Duration, existingRegistration?: Registration): RegistrationPriceQuote;
-  
-  canRenew(name: ENSName, atTimestamp: Timestamp, duration: Duration, existingRegistration: Registration): boolean;
-  getRenewalPriceQuote(name: ENSName, atTimestamp: Timestamp, duration: Duration, existingRegistration: Registration): RenewalPriceQuote;
+  canRegister(
+    name: ENSName,
+    atTimestamp: Timestamp,
+    duration: Duration,
+    existingRegistration?: Registration,
+  ): boolean;
+  getRegistrationPriceQuote(
+    name: ENSName,
+    atTimestamp: Timestamp,
+    duration: Duration,
+    existingRegistration?: Registration,
+  ): RegistrationPriceQuote;
+
+  canRenew(
+    name: ENSName,
+    atTimestamp: Timestamp,
+    duration: Duration,
+    existingRegistration: Registration,
+  ): boolean;
+  getRenewalPriceQuote(
+    name: ENSName,
+    atTimestamp: Timestamp,
+    duration: Duration,
+    existingRegistration: Registration,
+  ): RenewalPriceQuote;
 }
 
+// Allows for trustless on-chain name registration
 export interface OnchainRegistrar extends Registrar {
   getOnchainRegistrar(): ContractRef;
 }
 
+// Allows for trustless off-chain name registration
 export interface OffchainRegistrar extends Registrar {
   getOnchainRegistrar(): null;
 }
 
 /**
- * Identifies that a name was passed to a `Registrar` function that was not relevant for that `Registrar`.
- * 
- * @param message a reason why `name` was NOT relevant for `Registrar`.
- * @param label the `ENSName` value that was NOT relevant.
+ * Identifies that a name was passed to a `Registrar` function that is not managed by that `Registrar`.
```

Comment:
@FrancoAguzzi I've updated the terminology "managed by" to "issued by". In the context of a `Registrar` we should generally use the terminology "issued by".

I then further refined this from "issued by" to "issuable" because "issued by" suggests that the subname has already been issued, but what matters is that the `Registrar` has the capability to issue.

Reply:
Exact 🎯